src/Passes/Makefile

```diff
@@ -0,0 +1,7 @@
+nothing:
```

Comment:
The other makefile which only cleans doesn't have this protection

Reply:
Any Makefile that only cleans should have this protection imo and it was a mistake that some did not. It is corrected now.

Note: The only purpose of the root Makefile is to perform cleans at the moment. Arguably this functionality should be moved to `./manage` which does all of the other build and test related tasks. Likely this Makefile will be deprecated in the future.